velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -98,6 +98,35 @@ struct WeekFunction : public InitSessionTimezone<T> {
   }
 };
 
+template <typename T>
+struct YearOfWeekFunction : public InitSessionTimezone<T> {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE int32_t computeYearOfWeek(const std::tm& dateTime) {
```

Comment:
this function is used only once in a method that doesn't have anything else; let's fold it into the caller.